security/kaspersky_server.ksy

```diff
@@ -0,0 +1,13 @@
+meta:
+  id: kasperskyserver
+  endian: le
+  title: Kaspersky for Windows Server quarantine file parser
+  license: CC-BY-SA-4.0
+  ks-version: 0.9
+doc: |
+  Creator: Florian Bausch, ERNW Research GmbH, https://ernw-research.de
+  License: CC-BY-SA-4.0 https://creativecommons.org/licenses/by-sa/4.0/
```

Comment:
Not a very suitable license for something that is simultaneously docs and code.

Reply:
> Not a very suitable license for something that is simultaneously docs and code.

I agree. See also https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software.